### PR TITLE
Added support for macOS 10.9

### PR DESCRIPTION
**Pull request by OleksandStepanov**
Hello @skreutzberger,

We would like to use your awesome logging solution in our project, but we have minimum deployment target 10.9 for macOS app. In this PR I'm just fixing compile issues for 10.9 using `#if available` Swift structure.
SPM support starts from 10.10, so this change will affect only those who uses your framework via CocoaPods or git submodules. 
Hope it makes sense.

With regards.

### CODE REVIEW

File: Sources/SBPlatformDestination.swift [hunk 1]
```diff
@@ -468,11 +470,19 @@ public class SBPlatformDestination: BaseDestination {
...
 
         details["os"] = OS
-        let osVersion = ProcessInfo.processInfo.operatingSystemVersion
-        // becomes for example 10.11.2 for El Capitan
-        var osVersionStr = String(osVersion.majorVersion)
-        osVersionStr += "." + String(osVersion.minorVersion)
-        osVersionStr += "." + String(osVersion.patchVersion)
+        
+        var osVersionStr: String
+        if #available(OSX 10.10, *) {
```
Comment (skreutzberger) [inline Sources/SBPlatformDestination.swift]: why is this necessary? what would be if the OS is iOS?

Comment (OleksandStepanov) [reply Sources/SBPlatformDestination.swift]: `ProccessInfo.operatingSystemVersion` is available only starting macOS 10.10. Asterisk after the comma means that this check will pass on all other platforms according to [the documentation](https://docs.swift.org/swift-book/ReferenceManual/Attributes.html). Because minimum support for macOS is 10.9, this test will not pass only for this version and only on macOS, that's why it's safe to put "10.9" into `osVersionStr`. We won't know the patch version, but I believe that's not critical.

**Comment by OleksandStepanov** [comment]
Hello @skreutzberger,
Could you provide any feedback on this PR, please?

File: Sources/SBPlatformDestination.swift (hunk 1 above)
Comment (skreutzberger) [reply Sources/SBPlatformDestination.swift]: I am unsure about that condition really works as intended and am concerned that we will break support for newer OSX versions and general iOS and Apple TV versions. I checked the Apple docs (thanks for the link!) but they did not make me smarter on that one.

Did you try out the condition and does it really work?

**Comment by skreutzberger** [comment]
won't do
